src/quest_manager/views.py

```diff
@@ -214,6 +214,18 @@ def get_context_data(self, **kwargs):
         return context
 
 
+def categorydetail(request, category_id):
+
```

Comment:
This should be a CBV (Class based view) and it should be with the rest of this model's views (CategoryUpdate, CategoryDelete, CategoryCreate... etc)